etc/adminhtml/system.xml

```diff
@@ -73,6 +73,24 @@
                         <field id="debug">1</field>
                     </depends>
                 </field>
+                <field id="sandbox" translate="label" sortOrder="12" type="select" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable Sandbox</label>
+                    <comment>Change all API requests to be made to TaxJar's sandbox account.</comment>
```

Comment:
@ryanreeves-taxjar Since our sandbox environment primarily supports API calculations at the moment, should we consider saying something like:

```
Use the TaxJar sandbox environment to test API sales tax calculations.
```

Moreover, what if address validation is enabled? We don't support AV in the sandbox. For transaction sync we'll return stubbed responses.

- Do we need to show a notice / notification in admin when sandbox is enabled?
- Do we need to disable or lock specific functionality such as address validation or syncing nexus addresses when sandbox is enabled?
- Do we need to explain any limitations around the sandbox environment elsewhere or link to our API documentation?

Reply:
@fastdivision I've had several customers confused by your third bullet point as well. I definitely think we need a better strategy here, for at least the messaging aspect.

Reply:
For address validation, specifically, could we just use their regular API token (if it's still sitting in the other field) even when Sandbox is enabled? That way, there's no visible problem to the customer, unless they don't have a production token in place.

Reply:
After some review and internal discussions:  

- We'll display a warning message inside Magento's admin to notify the user that the sandbox is enabled.
- We're allowing all functionality.  All API endpionts have been confirmed to either work or fail gracefully.

Reply:
@ryanreeves-taxjar Can we update the description here to include some more detail similar to the new notification?

```
Use the TaxJar sandbox environment to test API sales tax calculations. We recommend only using this mode in a staging environment. Some features may not work as expected.
```